tck-dist/src/main/asciidoc/sections/07.2ee-mode.adoc

```diff
@@ -59,7 +59,21 @@ To enable this extension include the following configuration to your surefire pl
 include::{starterLoc}/ee-pom.xml[tags=displayName]
 ----
 
-include::07.aentity-modes.adoc[]
+==== Entity Modes
+
+There are situations where we need to test entities from a specific provider, 
+and other times where the test is generalized to work for any provider.
+
+To accomplish this tests are annotated with either `@Persistence`, `@NoSQL`, or no annotation.
```

Comment:
Having both annotations would give the same behavior as no annotation so either option is possible.

Reply:
While it is possible that for some providers these might result in the same behavior, there is no requirement currently in the specification for providers to support unannotated entities.  Unless that changes, we do need to include the annotations so that all NoSQL and Jakarta-persistence based providers can pass the TCK.

Reply:
Okay, I've fixed how this is documented to avoid confusion.